src/ActiveJobsList.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
-import JobDescription from './JobDescription'
 
 import './ActiveJobsList.css'
 import abortAllIcon from './assets/images/abort_all.svg'
 import doneIcon from './assets/images/done.svg'
 
+const getJobDescription = require('./getJobDescription.js')
```

Comment:
Let's rename the module to `job_description.js`

Reply:
I forget, does `import` not work in this case?

Reply:
It works, using import ... from ... now.